modular_skyrat/modules/modular_items/code/recipes_misc.dm

```diff
@@ -50,3 +50,12 @@
 				/obj/item/stack/sheet/cloth = 2)
 	time = 80
 	category = CAT_MISC
+
+/datum/crafting_recipe/makeshift/chisel //BUBBERSTATION ADDITION
+	name = "Makeshift Chisel"
```

Comment:
why is this makeshift, theres no difference in the output

Reply:
Well true